tlspuffin/src/boringssl/deterministic.rs

```diff
@@ -0,0 +1,42 @@
+use boringssl_sys::RAND_reset_for_fuzzing;
+
+/// Reset BoringSSL PRNG
+pub fn reset_rand() {
+    unsafe {
+        RAND_reset_for_fuzzing();
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use std::fmt::format;
+
+    use puffin::{
+        put::PutOptions,
+        trace::{Action, InputAction, OutputAction, Step, Trace, TraceContext},
+    };
+
+    use crate::{
+        boringssl::deterministic::reset_rand,
+        put_registry::TLS_PUT_REGISTRY,
+        tls::{
+            seeds::{create_corpus, seed_client_attacker_full_boring},
+            trace_helper::TraceHelper,
+        },
+    };
+
+    // BUG: This test only works in a single threaded cargo test execution
+    #[ignore]
```

Comment:
Add a TODO comment here, as we use that one typically in the codebase to mark todos.

Maybe this is even worth a task on github.

Reply:
Done